mysql-test/suite/rpl/t/rpl_mdev12179.test

```diff
@@ -70,9 +68,7 @@ SELECT * FROM t1 ORDER BY a;
 # Let the slave mysqld server start again.
 # As we are restarting, also take the opportunity to test --gtid-pos-auto-engines
 --echo *** Restart server with --gtid-pos-auto-engines=innodb,myisam ***
---append_file $MYSQLTEST_VARDIR/tmp/mysqld.2.expect
```

Comment:
I do not think overwrite here was desired, rather an "append". The file "must" be there, it might contain "wait"